docs/DeveloperGuide.md

```diff
@@ -449,11 +449,28 @@ Use case ends.
 
 1.  User wants to delete an order.
 2.  User chooses to <ins>view all orders of a specified customer from contacts list (UC04)</ins>.
-3.  User deletes the customer by the customer name and the order index.
+3.  User deletes the order by the customer name and the order index.
 4.  SellSavvy updates the status of the order and indicates that the action is successful.
 
 Use case ends.
 
+**Use case 8: Filter order list by order status**
```

Comment:
![image](https://github.com/user-attachments/assets/80bffcfc-55fe-4306-b7f9-ca8ab426a81e)

i not sure but can we move it above? to below listorder then those below we can change to chooses to view all orders or filtered orders. can someone help on this? :(

Reply:
Nvm u can ignore this first not ipt